homeassistant/helpers/entity_values.py

```diff
@@ -0,0 +1,42 @@
+"""A class to hold entity values."""
+import fnmatch
+import re
+
+from homeassistant.core import split_entity_id
+
+
+class EntityValues(object):
+    """Class to store entity id based values."""
+
+    def __init__(self, exact=None, domain=None, glob=None):
+        """Initialize an EntityConfigDict."""
+        self._cache = {}
+        self._exact = exact
+        self._domain = domain
+
+        if glob is not None:
+            glob = {re.compile(fnmatch.translate(key)): value
+                    for key, value in glob.items()}
+
+        self._glob = glob
+
+    def get(self, entity_id):
```

Comment:
I think the order should be glob that will be overwritten with domain and hight level is entity customize. We have only one level why we neef a deep stuff @andrey-git?